buildstockbatch/test/test_results/reference_files/results_job1.json

```diff
@@ -0,0 +1,1480 @@
+        "SimulationOutputReport.electricity_bath_fan_kwh": 443.5247408840211,
+        "SimulationOutputReport.electricity_ceiling_fan_kwh": 6650.720920571772,
+        "SimulationOutputReport.electricity_extra_refrigerator_kwh": 0.0,
+        "SimulationOutputReport.electricity_freezer_kwh": 0.0,
+        "SimulationOutputReport.electricity_pool_heater_kwh": 0.0,
+        "SimulationOutputReport.natural_gas_pool_heater_therm": 0.0,
+        "SimulationOutputReport.electricity_pool_pump_kwh": 0.0,
+        "SimulationOutputReport.electricity_hot_tub_heater_kwh": 0.0,
+        "SimulationOutputReport.natural_gas_hot_tub_heater_therm": 0.0,
+        "SimulationOutputReport.electricity_hot_tub_pump_kwh": 0.0,
+        "SimulationOutputReport.natural_gas_grill_therm": 0.0,
+        "SimulationOutputReport.natural_gas_lighting_therm": 0.0,
+        "SimulationOutputReport.natural_gas_fireplace_therm": 0.0,
+        "SimulationOutputReport.electricity_well_pump_kwh": 0.0,
+        "SimulationOutputReport.electricity_recirc_pump_kwh": 0.0,
+        "SimulationOutputReport.electricity_vehicle_kwh": 0.0,
+        "SimulationOutputReport.weight": 1.0,
+        "SimulationOutputReport.wall_area_above_grade_conditioned_ft_2": 21724.44,
+        "SimulationOutputReport.wall_area_above_grade_exterior_ft_2": 22446.75,
+        "SimulationOutputReport.wall_area_below_grade_ft_2": 0.0,
+        "SimulationOutputReport.floor_area_conditioned_ft_2": 61452.0,
+        "SimulationOutputReport.floor_area_attic_ft_2": 0.0,
+        "SimulationOutputReport.floor_area_lighting_ft_2": 61452.0,
+        "SimulationOutputReport.roof_area_ft_2": 34799.33,
+        "SimulationOutputReport.window_area_ft_2": 6517.33,
+        "SimulationOutputReport.door_area_ft_2": 200.0,
+        "SimulationOutputReport.duct_surface_area_ft_2": 19664.64,
+        "SimulationOutputReport.size_heating_system_kbtu_h": 541.01,
+        "SimulationOutputReport.size_cooling_system_kbtu_h": 7175.16,
+        "SimulationOutputReport.size_water_heater_gal": 1620.0,
+        "SimulationOutputReport.upgrade_name": "",
+        "SimulationOutputReport.upgrade_cost_usd": "",
+        "SimulationOutputReport.applicable": true,
+        "building_id": 1,
+        "started_at": "20201008T032605Z",
+        "completed_at": "20201008T033246Z",
+        "completed_status": "Success",
+        "upgrade": 0
+    },
+    {
+        "started_at": "20201008T021200Z",
+        "completed_at": "20201008T021318Z",
+        "completed_status": "Fail",
+        "building_id": 2,
+        "step_failures": [
+            {
+                "measure_dir_name": "BuildExistingModel",
+                "step_errors": [
+                    "Cannot find weather file at /var/simdata/openstudio/lib/resources/measures/HPXMLtoOpenStudio/resources/var/simdata/openstudio/weather/USA_TX_Houston-Bush.Intercontinental.AP.722430.epw."
+                ]
+            }
```

Comment:
This `step_failures` entry is the only change to this file, other than unzipping it.